package.json

```diff
@@ -249,7 +249,6 @@
 		"playground:build": "npm run storybook:build",
 		"playground:dev": "echo \"Please use storybook:dev instead.\"",
 		"preenv": "npm run check-engines",
-		"env": "wp-scripts env",
```

Comment:
Should we provide a deprecation message when someone tries to run `wp-scripts env`? Could just keep `packages/scripts/scripts/env.js` with a single line that logs a helpful deprecation message

Reply:
Done!